client/components/telecom/pack/migration/pack-migration-process.service.js

```diff
@@ -198,7 +207,9 @@ angular.module('managerApp').service('PackMigrationProcess', function ($q, OvhAp
 
   self.selectOffer = function (offer) {
     migrationProcess.selectedOffer = offer;
-    if (migrationProcess.selectedOffer.totalSubServiceToDelete > 0) {
+    if (migrationProcess.selectedOffer.offerName.toLowerCase().includes('ftth')) {
```

Comment:
prefer using `_.includes` as native js `includes` is not supported by all browsers